Dockerfile.alpine

```diff
@@ -17,9 +17,12 @@ RUN apk add --no-cache bzip2 ca-certificates coreutils libstdc++ tini w3m &&\
   apk del bzip2 w3m
 
 # create user, group, and set permissions
-RUN adduser -u 503 -g 503 -h ${TS_DIRECTORY} -D teamspeak &&\
-  mkdir /data &&\
-  chown -R teamspeak:teamspeak ${TS_DIRECTORY} /data
+RUN  echo "**** create teamspeak user and make our folders ****" && \
+ groupmod -g 1000 users && \
```

Comment:
Just curious on the benefit of adding a users group.  I personally try to stay away from UIDs and GIDs that really exist on typical systems (e.g. - `1000`) since the process's user (or group) in the container would be the same as outside the container.